### PR TITLE
Sync with Symfony SE

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -14,7 +14,6 @@ public function registerBundles()
             new Symfony\Bundle\TwigBundle\TwigBundle(),
             new Symfony\Bundle\MonologBundle\MonologBundle(),
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
-            new Symfony\Bundle\AsseticBundle\AsseticBundle(),
```
Comment (WouterJ) [inline app/AppKernel.php]: assetic is removed from core SE. I think the only place we require assetic is with the CreateBundle, which isn't used in the standard edition.

Maybe it's a better idea to remove the Assetic dep in CreateBundle.

**Comment by lsmith77** [comment]
needs a rebase it seems

**Comment by WouterJ** [comment]
Done.

**Comment by lsmith77** [comment]
hmm seems like we have an issue with the DBAL/ORM version ..

**Comment by WouterJ** [comment]
closing in favor of https://github.com/symfony-cmf/standard-edition/pull/46

**Pull request closed without merging**